commands/build_lint.js

```diff
@@ -120,25 +130,23 @@ module.exports = function ({ output, lang, timeout }, sourceDir, { lint } = { li
         return log(log.chalk.bold(log.symbols.ok + ' Presets are valid'))
       }
 
-      var pack = tar.pack()
-      pack.on('error', done)
-      pack.entry({ name: 'presets.json' }, stringify(presets))
-      pack.entry({ name: 'translations.json' }, stringify(translations))
+      safelyZipBuffer(stringify(presets), 'presets.json')
```

Comment:
This calls `zip.addBuffer` with a string, not a Buffer. Should we make sure to convert the argument to a Buffer before passing it to `yazl`?

Reply:
I correct myself. `zip.addBuffer` receives any. But safer to only pass a buffer I think